cmd_cmd.go

```diff
@@ -59,7 +60,18 @@ will be launched via "bash -c" using "exec".`,
 			rand.Read(spanBytes)
 
 			start := time.Now()
-			err := runCommand(subcmd)
+
+			// copy out the current set of fields to avoid later modification
+			localFields := map[string]interface{}{}
```

Comment:
Minor: create a variable for the spanID so we don't need to fmt.Sprintf twice 😄

Reply:
good call. fixed that.